xbmc/pvr/PVRDatabase.cpp

```diff
@@ -45,7 +45,7 @@ void CPVRDatabase::CreateTables()
 {
   CLog::Log(LOGINFO, "PVR - %s - creating tables", __FUNCTION__);
 
-  CLog::Log(LOGDEBUG, "PVR - %s - creating table 'channels'", __FUNCTION__);
+  CLog::Log(LOGDEBUG, "CPVRDatabase - %s - creating table 'channels'", __FUNCTION__);
```

Comment:
yes, sure, how else? There are about 50 files or more in the pvr folder, some are in dbwrapper, how should I know? Pls. refer to the dvdrecoder section and lok how DEBUGLOG are written - I think its 90% classname.

Before I dived into pvr I had to look for some decoding issues (ticket will follow for this), and it was so much easier to find the right files.

To not spam normal users I only changed naming conventions in LOGDEBUG, I left all others untouched.